tests/test/dashboard.test.js

```diff
@@ -1,12 +1,21 @@
 const url = require('../helpers/url').main
+const wait = require('../helpers/wait')
+
+/**
+ * Helper function to list locks on the dashboard
+ */
+const listLocksOnPage = async () =>
+  page.evaluate(() => {
+    const lockElements = document.getElementsByClassName('lock')
+    return Array.from(lockElements).map(el => el.getAttribute('data-address'))
+  })
 
 describe('The Unlock Dashboard', () => {
   beforeAll(async () => {
-    await page.goto(url('/dashboard'))
+    await page.goto(url('/dashboard'), { waitUntil: 'networkidle2' })
+    await wait.forLoadingDone()
   })
 
-  const testLockAddress = '0x5Cd3FC283c42B4d5083dbA4a6bE5ac58fC0f0267'
```

Comment:
Not used anymore. What we do now is that we first look at the locks on page, and then after deploying a new one, identify which is new.